extension/app/src/lib/auth.ts

```diff
@@ -85,5 +88,5 @@ const fetchMe = async (token: string): Promise<UserTypeWithWorkspaces> => {
     );
   }
   const me = await response.json();
-  return me.user;
```

Comment:
Just for info, we did not caught it during review, this broke the extension which is still calling /me with a fetch 🙈 
(fixed now)